nixos/modules/services/misc/autobrr.nix

```
@@ -0,0 +1,106 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+
+let
+  cfg = config.services.autobrr;
+in
+{
+  options = {
+    services.autobrr = {
+      enable = mkEnableOption (lib.mdDoc "Autobrr");
+
```

I'd consider adding a `package` option, which is more and more common.

```suggestion
      enable = mkEnableOption (lib.mdDoc "Autobrr");

      package = mkPackageOption pkgs "autobrr" { };
```